docs/templates/tutorials/distributed-tuning.md

```diff
@@ -0,0 +1,122 @@
+# Distributed Tuning
+
+Keras Tuner makes it easy to perform distributed hyperparameter search. No changes to your code are needed to scale up from running single-threaded locally to running on dozens or hundreds of workers in parallel. Distributed Keras Tuner uses a chief-worker model. The chief runs a service to which the workers report results and query for the hyperparameters to try next. The chief should be run on a single-CPU worker (or alternatively as a separate process on one of the workers).
+
+### Configuring distributed mode
+
+Configuring distributed mode for Keras Tuner only requires setting three environment variables:
+
+**KERASTUNER_TUNER_ID**: This should be set to "chief" for the chief process. Other workers should be passed a unique ID (by convention, "tuner0", "tuner1", etc).
+
+**KERASTUNER_ORACLE_IP**: The IP address of the chief service.
```

Comment:
Please provide a bit more context (keeping in mind that users reading this will likely be in the process of launching up a set of instances on AWS or GCP)

Reply:
Done